spec/unit/controllers/services/service_instances_controller_spec.rb

```diff
@@ -885,6 +885,53 @@ def stub_delete_and_return(status, body)
             expect(last_response.status).to eq(400)
             expect(decoded_response['code']).to eq(60002)
           end
+
+          context 'when a service instance share exists between spaces' do
```

Comment:
We got a bit confused initially reading through this because of the var name `space` since there's more than one space being referred to. We thought the var name `source_space` was great, so perhaps something like `target_space` would be more descriptive as well.

Reply:
Hey @anniesing 

Thanks for noticing this, we totally agree that it would make more sense to refer to `target_space`.

Would you accept a PR later that addresses this? We are happy to create a chore in our backlog to look at this, and other test files we have changed, that might have vague variable names. 

Thanks, 

Sapi, (@deniseyu, alex)

Reply:
A future PR sounds great.